popSimManu.tex

```diff
@@ -145,15 +145,24 @@ \section*{Introduction}
 benchmark the quality of inferences gleaned from computational population genetics.
 As large databases of population genetic variation begin to be used to inform public health procedures,
 the accuracy and quality of these inferences is becoming ever more important.
+%ALG: cite large databases of population genetic variate to inform public health procedures? UK biobank, 23andMe???
 %CK: this first paragraph is super long - might be helpful to break up
+%ALG: I think breaking up here is fine.
```

Comment:
It's OK now - the only reason it's long is the extensive citation list (which I don't love but am leaving in...).